src/h5web/visualizations/heatmap/HeatmapVis.module.css

```diff
@@ -90,3 +90,15 @@
   text-anchor: inherit;
   dominant-baseline: inherit;
 }
+
+.tooltip {
+  color: inherit !important;
+  background-color: rgba(245, 251, 239, 0.7) !important;
```

Comment:
Why are the `!important` needed ?

Reply:
It overrides the inline styles set by VX.